finetune_adapter.py

```diff
@@ -154,7 +151,7 @@ def validate(fabric: L.Fabric, model: torch.nn.Module, val_data: np.ndarray, tok
     fabric.print(instruction)
     sample = {"instruction": instruction, "input": ""}
     prompt = generate_prompt(sample)
-    encoded = tokenizer.encode(prompt, bos=True, device=model.device)
```

Comment:
The bos token doesn't seem to be necessary. The loss convergence seems to be unaffected though.

Reply:
Can we remove the `bos` and `eos` tokenizer logic to simplify? AFAIK it's now unused

Reply:
It is unclear how we would justify removing that. Maybe the bos yes if we are sure that it won't be used. But we should probably do that in a follow up not here, because here I'm just fixing basic stuff for the existing finetuning script.

Reply:
I'm okay with doing this separately.